lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1363,6 +1443,108 @@ bool GrandCentralPass::traverseField(
       .Default([](auto a) { return true; });
 }
 
+bool GrandCentralPass::traverseViewField(
+    Attribute field, VerbatimBuilder &path,
+    SmallVector<VerbatimXMRbuilder> &xmrElems,
+    SmallVector<InterfaceElemsBuilder> &interfaceBuilder, ViewIntrinsicOp view,
+    size_t &idx) {
+  return TypeSwitch<Attribute, bool>(field)
+      .Case<AugmentedGroundTypeAttr>([&](AugmentedGroundTypeAttr ground) {
+        // Grab next signal, increment index counter.
+        auto index = idx++;
+        if (index >= view.getNumOperands()) {
+          view.emitError("more ground types found than view has operands");
+          return false;
+        }
```

Comment:
Once we iterate (after this PR lands) on the Chisel/FIRRTL intrinsic representation for this, we can move this into a verifier. I'm good with keeping this in the pass for now and, as you see already everywhere, there is tons of verification that only happens during this pass (but is extremely rare to hit as Chisel won't screw up the annotation / intrinsic representation).